ast/builtins.go

```diff
@@ -68,6 +68,18 @@ var DefaultBuiltins = [...]*Builtin{
 
 	//Sort
 	Sort,
+
+	//Type
+	IsNumber,
+	IsString,
+	IsBoolean,
+	IsArray,
+	IsSet,
+	IsObject,
+	IsNull,
+
+	//Type Name
+	TypeNameVar,
```

Comment:
Nit: `TypeNameVar` => `TypeNameBuiltin` to be consistent with the `walk` built-in function (see above.)